pdfly/cli.py

```diff
@@ -104,6 +104,32 @@ def cat(
     pdfly.cat.main(filename, fn_pgrgs, output, verbose)
 
 
+@entry_point.command(name="rm")  # type: ignore[misc]
```

Comment:
There is one last thing missing: some `help` documentation, that should be visible when calling `pdfly rm --help`

Reply:
To maintain code consistency, I could create a new script, 'rm.py', with a main function that references the main function in 'cat.py', if you agree

Reply:
I have added a new module implementing this

Reply:
Great, good job! 👍